script/release/update-version-all.sh

```diff
@@ -119,6 +119,11 @@ for repository in `cat ${scriptDir}/../repository-list.txt` ; do
             mvn -B -s $settingsXmlFile clean install -DskipTests
             returnCode=$?
 
+        elif [ "$repository" = "drlx-parser" ];then
+            #update version in drlx-parser/pom.xml
+            cd drlx-parser
+            mvn versions:set -DnewVersion=$newVersion
```

Comment:
Would it make sense to call the function `mvnVersionsSet()`? It already has all the common config needed.

Reply:
YES. Done